chat/management/commands/rundiscordbot.py

```diff
@@ -61,36 +63,43 @@ async def send_puzzles_stuck(message):
         Puzzle.objects.filter(
             Q(hunt=settings.BOT_ACTIVE_HUNT),
             Q(status=Puzzle.STUCK) | Q(status=Puzzle.EXTRACTION),
-        )
+        ).select_related("chat_room")
     )
     await send_puzzles(message, puzzles, "Stuck puzzles")
 
 
 async def send_puzzles(message, puzzles, title):
     print(f"Sending puzzles {puzzles}")
-    embed = discord.Embed()
+    embed = discord.Embed(title=title)
     lines = []
     for p in puzzles:
-        line = "- "
+        line_title = ""
         if p.is_solved():
-            line += f"[{p.answer}] "
-        line += f"[{p.name}]({p.url})" if p.url else p.name
+            line_title += f"[{p.answer}] "
+        line_title += p.name
+
+        line = ""
+        if p.url:
+            line += f"[Puzzle]({p.url}) "
         if p.sheet:
-            line += f" ([sheet]({p.sheet}))"
-        lines.append(line)
+            line += f"([sheet](https://smallboard.app/puzzles/s/{p.id}))"
+        if p.chat_room:
+            if p.chat_room.text_channel_url:
+                line += f"([chat]({p.chat_room.text_channel_url}))"
+
+        lines.append((line_title, line))
     print(f"lines: {lines}")
```

Comment:
looks like a debugging line that can be removed

Reply:
Yeah there were a couple of these, I thought maybe they were intentionally left in? @akirabaruah

Reply:
They were, but I realize now that they're kinda excessive. We should probably be a bit more principled with logging in this bot. Open to suggestions 👍